renderdoc/driver/gl/gl_common.cpp

```diff
@@ -415,6 +415,8 @@ void CheckExtensions(const GLHookSet &gl)
       if(ext == NULL || !ext[0] || !ext[1] || !ext[2] || !ext[3])
         continue;
 
+      RDCLOG("glGetStringi(eGL_EXTENSIONS, %i) = %s", i, ext);
```

Comment:
Oh I didn't realize there is already a list of extensions printed on replay. I added this because sometimes capture crashes and it's usually because of an un-exposed extension.